### DISCUSSION TITLE
Hello everyone!!!

### DISCUSSION THREAD

**Discussion by yashaswya** [OpenMined/courses#56] [Introduce Yourself]
Hey everyone,

I'm Yashaswy from Limerick, Ireland originally from Hyderabad, India. Excited to start this journey and learn everything this course has to offer. Can't wait to get started.

**Comment by yemikifouly** [comment]
Hello Yashaswy! 🙂 Welcome to the discussion board! Have you checked out the course yet? I hope you enjoy it. Also, if you haven't attended our welcoming event, consider watching the recording [here](https://www.youtube.com/watch?v=ZdfpXIyY9iY). We answer some questions that you might find helpful. 🙂
